apps/site/assets/ts/stop/__tests__/DisplayTimeTest.tsx

```diff
@@ -144,5 +161,22 @@ describe("DisplayTime", () => {
       expect(screen.findByText("Delayed 11:43 AM")).toBeTruthy();
       expect(screen.getByText("11:38 AM")).toHaveClass("strikethrough");
     });
+    it("(for subway) shows only predicted time (1+ hour away)", () => {
```

Comment:
Is this just testing an existing behavior that predictions an hour or more out show for subway only?

Reply:
ah the title's probably a bit confusing.. it's testing a new behavior, but I probably could've combined this with the other (<1 hour away) test